lib/v2/screens/profile_screens/security/security_screen.dart

```diff
@@ -67,14 +71,31 @@ class SecurityScreen extends StatelessWidget {
                         icon: const Icon(Icons.update),
                         title: 'Export Private Key'.i18n,
                         description: 'Export your private key so you can easily recover and access your account.'.i18n,
-                        onTap: () {},
+                        onTap: () => Share.share(settingsStorage.privateKey),
                       ),
-                      SecurityCard(
-                        icon: SvgPicture.asset('assets/images/key_guardians_icon.svg'),
-                        title: 'Key Guardians'.i18n,
-                        description:
-                            'Choose 3 - 5 friends and/or family members to help you recover your account in case.'.i18n,
-                        onTap: () {},
+                      BlocBuilder<SecurityBloc, SecurityState>(
+                        buildWhen: (previous, current) =>
```

Comment:
This is a bug. 
If the current.hasNotification is false, this will not be redrawn. 

Which means, No matter what,  the icon will never go away.

Reply:
Unless you navigate away and come back.